app/Http/Controllers/OAuthController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Laravel\Socialite\Facades\Socialite;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+
+class OAuthController extends Controller
+{
+    public function redirect($socialPlatform) {
```

Comment:
can you add comment every function like this

/**
 * Redirect QAuth login
 * 
 * @param string $socialPlatform
 * @param response
 */
public function redirect($socialPlatform)
{
}